{"text": "2024-09-24 06:31:01 | INFO    | __main__:stash:131 | run - {'image': '20231121', 'version': '0.10.1.post2', 'role': 'EpicPlayer', 'headless': True}\n", "record": {"elapsed": {"repr": "0:00:01.129956", "seconds": 1.129956}, "exception": null, "extra": {"image": "20231121", "version": "0.10.1.post2", "role": "EpicPlayer", "headless": true}, "file": {"name": "claim.py", "path": "/home/runner/work/epic_N/epic_N/epic/src/claim.py"}, "function": "stash", "level": {"icon": "ℹ️", "name": "INFO", "no": 20}, "line": 131, "message": "run", "module": "claim", "name": "__main__", "process": {"id": 3663, "name": "MainProcess"}, "thread": {"id": 140102531357568, "name": "MainThread"}, "time": {"repr": "2024-09-24 06:31:01.202273+08:00", "timestamp": 1727130661.202273}}}
{"text": "2024-09-24 06:31:09 | INFO    | __main__:claim_epic_games:97 | Try to flush cookie - {'task': 'claim_epic_games'}\n", "record": {"elapsed": {"repr": "0:00:09.298395", "seconds": 9.298395}, "exception": null, "extra": {"task": "claim_epic_games"}, "file": {"name": "claim.py", "path": "/home/runner/work/epic_N/epic_N/epic/src/claim.py"}, "function": "claim_epic_games", "level": {"icon": "ℹ️", "name": "INFO", "no": 20}, "line": 97, "message": "Try to flush cookie", "module": "claim", "name": "__main__", "process": {"id": 3663, "name": "MainProcess"}, "thread": {"id": 140102531357568, "name": "MainThread"}, "time": {"repr": "2024-09-24 06:31:09.370712+08:00", "timestamp": 1727130669.370712}}}
{"text": "2024-09-24 06:31:39 | WARNING    | epic_games.agent:authorize:266 | 执行超时 - {'task': 'authorize', 'retry': 0}\n", "record": {"elapsed": {"repr": "0:00:39.507807", "seconds": 39.507807}, "exception": null, "extra": {"task": "authorize", "retry": 0}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_N/epic_N/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "⚠️", "name": "WARNING", "no": 30}, "line": 266, "message": "执行超时", "module": "agent", "name": "epic_games.agent", "process": {"id": 3663, "name": "MainProcess"}, "thread": {"id": 140102531357568, "name": "MainThread"}, "time": {"repr": "2024-09-24 06:31:39.580124+08:00", "timestamp": 1727130699.580124}}}
{"text": "2024-09-24 06:32:09 | WARNING    | epic_games.agent:authorize:266 | 执行超时 - {'task': 'authorize', 'retry': 1}\n", "record": {"elapsed": {"repr": "0:01:09.509319", "seconds": 69.509319}, "exception": null, "extra": {"task": "authorize", "retry": 1}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_N/epic_N/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "⚠️", "name": "WARNING", "no": 30}, "line": 266, "message": "执行超时", "module": "agent", "name": "epic_games.agent", "process": {"id": 3663, "name": "MainProcess"}, "thread": {"id": 140102531357568, "name": "MainThread"}, "time": {"repr": "2024-09-24 06:32:09.581636+08:00", "timestamp": 1727130729.581636}}}
{"text": "2024-09-24 06:32:12 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games&__cf_chl_rt_tk=wnhm_3QMGiXPvkIm0EWMumLXIGf1ToA3C6ZuHJt7pTg-1727130731-0.0.1.1-6804'}\n", "record": {"elapsed": {"repr": "0:01:11.970354", "seconds": 71.970354}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games&__cf_chl_rt_tk=wnhm_3QMGiXPvkIm0EWMumLXIGf1ToA3C6ZuHJt7pTg-1727130731-0.0.1.1-6804"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_N/epic_N/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "ℹ️", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3663, "name": "MainProcess"}, "thread": {"id": 140102531357568, "name": "MainThread"}, "time": {"repr": "2024-09-24 06:32:12.042671+08:00", "timestamp": 1727130732.042671}}}
{"text": "2024-09-24 06:32:42 | WARNING    | epic_games.agent:authorize:266 | 执行超时 - {'task': 'authorize', 'retry': 2}\n", "record": {"elapsed": {"repr": "0:01:41.972472", "seconds": 101.972472}, "exception": null, "extra": {"task": "authorize", "retry": 2}, "file": {"name": "agent.py", "path": "/home/runner/work/epic_N/epic_N/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "⚠️", "name": "WARNING", "no": 30}, "line": 266, "message": "执行超时", "module": "agent", "name": "epic_games.agent", "process": {"id": 3663, "name": "MainProcess"}, "thread": {"id": 140102531357568, "name": "MainThread"}, "time": {"repr": "2024-09-24 06:32:42.044789+08:00", "timestamp": 1727130762.044789}}}
{"text": "2024-09-24 06:32:42 | ERROR    | __main__:run:158 | An error has been caught in function 'run', process 'MainProcess' (3663), thread 'MainThread' (140102531357568): - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic_N/epic_N/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/home/runner/work/epic_N/epic_N/epic/src/claim.py\", line 158, in run\n    await agent.stash()\n  File \"/home/runner/work/epic_N/epic_N/epic/src/claim.py\", line 151, in stash\n    await self.claim_epic_games(context)\n  File \"/home/runner/work/epic_N/epic_N/epic/src/claim.py\", line 98, in claim_epic_games\n    if await epic.authorize(page):\n  File \"/home/runner/work/epic_N/epic_N/epic/src/epic_games/agent.py\", line 269, in authorize\n    raise RuntimeError(f\"Failed to flush token - agent={self.__class__.__name__}\")\nRuntimeError: Failed to flush token - agent=EpicGames\n", "record": {"elapsed": {"repr": "0:01:42.434825", "seconds": 102.434825}, "exception": {"type": "RuntimeError", "value": "Failed to flush token - agent=EpicGames", "traceback": true}, "extra": {}, "file": {"name": "claim.py", "path": "/home/runner/work/epic_N/epic_N/epic/src/claim.py"}, "function": "run", "level": {"icon": "❌", "name": "ERROR", "no": 40}, "line": 158, "message": "An error has been caught in function 'run', process 'MainProcess' (3663), thread 'MainThread' (140102531357568):", "module": "claim", "name": "__main__", "process": {"id": 3663, "name": "MainProcess"}, "thread": {"id": 140102531357568, "name": "MainThread"}, "time": {"repr": "2024-09-24 06:32:42.507142+08:00", "timestamp": 1727130762.507142}}}
